vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -370,7 +372,7 @@ func (d *VolumeDriver) Mount(r volume.MountRequest) volume.Response {
 	fstype := fs.FstypeDefault
 	isReadOnly := false
 	if err != nil {
-		_, err := d.decrRefCount(r.Name)
+		d.decrRefCount(r.Name)
```

Comment:
question about L368 - do you know why do we need Get() there? Docker does get before use, doesn't it cause double Get() ?